src/pages/docs/projects/custom-step-templates.md

```diff
@@ -112,32 +100,24 @@ It's important to note that you can only run script based custom step templates.
 
 To run a script based step template, perform the following.
 
-1. Navigate to **Library ➜ Step templates** area and click the **Run** button next to the script based custom step template or alternately, select a script template and click the **Run** button from the template editor page:
+1. Navigate to **Deploy ➜ Step templates** area and click the **Run** button next to the script based custom step template or alternately, select a script template and click the **Run** button from the template editor page:
 
    ![Run step template](/docs/projects/images/step-templates-run.png)
 
-2. Select a group of targets to run the step on. This can be done by target name or by environments and roles:
-
-   ![Select run targets](/docs/projects/images/step-templates-run-targets.png)
-
-3. Enter any required parameters:
+2. Select a group of targets to run the step on. This can be done by target name or by environments and tags.
+3. Enter any required parameters.
+4. Click the **Run now** button. This will execute the step as a new task. The full script can be found under the Template Parameters tab:
 
-   ![Enter parameter values](/docs/projects/images/step-templates-run-parameters.png)
```

Comment:
Removed as these fields should be apparent to the user (they're the only input fields on the page).